framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -4838,7 +4844,21 @@ protected final boolean shouldSkipUses(ExpressionTree exprTree) {
       return true;
     }
     Element elm = TreeUtils.elementFromTree(exprTree);
-    return checker.shouldSkipUses(elm);
+    boolean shouldSkipUses = checker.shouldSkipUses(elm);
+    if (showchecks) {
+      TypeMirror exprTypeMirror = TreeUtils.typeOf(exprTree);
+      System.out.printf(
+          "%s %s (at %s): actual tree = %s %s%n   expected: %s %s%n",
```

Comment:
Why is this labeled "expected"?  That doesn't seem to fit with `shouldSkipUses(ExpressionTree)`.